File: App-Encapsulamento-Java/app/src/main/java/com/example/pooencapsulamento/Livro.java
package com.example.pooencapsulamento;

public class Livro {
    private String titulo;
    private String autor;
    private String editora;
    private int paginas;
    private String paginas2;
    private String localizacao;

    public String getTitulo() {
        return titulo;
    }

    public void setTitulo(String titulo) {
        this.titulo = titulo;
    }

    public String getAutor() {
        return autor;
    }

    public void setAutor(String autor) {
        this.autor = autor;
    }

    public String getEditora() {
        return editora;
    }

    public void setEditora(String editora) {
        this.editora = editora;
    }

    public String getPaginas() {
        paginas2 = String.valueOf(paginas);
        return paginas2;
    }

    public void setPaginas(int paginas) {
        this.paginas = paginas;
    }

    public String getLocalizacao() {
        return localizacao;
    }

    public void setLocalizacao(String localizacao) {
        this.localizacao = localizacao;
    }
}

File: App-Encapsulamento-Kotlin/app/src/main/java/com/example/pookotlinencapsulamento/Livro.kt
package com.example.pookotlinencapsulamento

class Livro(
    private var _titulo: String,
    private var _autor: String,
    private var _editora: String,
    private var _paginas: String,
    private var _localizacao: String
) {
    var titulo: String
        get() = _titulo
        set(value) {
            _titulo = value
        }
    var autor: String
        get() = _autor
        set(value) {
            _autor = value;
        }
    var editora: String
        get() = _editora
        set(value) {
            _editora = value;
        }
    var paginas: String
        get() = _paginas
        set(value) {
            _paginas = value;
        }
    var localizacao: String
        get() = _localizacao
        set(value) {
            _localizacao = value;
        }
}